api/app/reviews/tests.py

```diff
@@ -1285,3 +1288,204 @@ def test_total_number_of_pages_when_zero(self):
+        db.session.commit()
+
+
+        closed_review = ReviewForm(2, datetime(2018, 4, 30))
+        closed_review.close()
+        review_forms = [
+            ReviewForm(1, datetime(2019, 4, 30)),
+            closed_review
+        ]
+        db.session.add_all(review_forms)
+        db.session.commit()
+
+        review_configs = [
+            ReviewConfiguration(review_form_id=review_forms[0].id, num_reviews_required=3, num_optional_reviews=0),
+            ReviewConfiguration(review_form_id=review_forms[1].id, num_reviews_required=3, num_optional_reviews=0)
+        ]
+        db.session.add_all(review_configs)
+        db.session.commit()
+
+        review_questions = [
+            ReviewQuestion(1, 1, None, None, 'multi-choice', None, None, True, 1, None, None, 0),
+            ReviewQuestion(1, 2, None, None, 'multi-choice', None, None, True, 2, None, None, 0),
+            ReviewQuestion(2, 3, None, None, 'multi-choice', None, None, True, 1, None, None, 0),
+            ReviewQuestion(2, 4, None, None, 'information', None, None, False, 2, None, None, 0)
+        ]
+        db.session.add_all(review_questions)
+        db.session.commit()
+
+        # Reference
+
+        self.test_response = Response(  # Nominating other
+            self.application_form.id, self.first_user_data.id)
+
+        self.add_to_db(self.test_response)
+        answers = [
+            # Answer(self.test_response.id, questions[0].id, 'other'),
+            # Answer(self.test_response.id, questions[1].id, 'Blah'),
+            Answer(self.test_response.id, questions[5].id, 'Mx'),
+            Answer(self.test_response.id, questions[6].id, 'Skittles'),
+            Answer(self.test_response.id, questions[7].id, 'Cat'),
+            Answer(self.test_response.id, questions[8].id, 'skittles@box.com'),
+        ]
+        db.session.add_all(answers)
+        db.session.commit()
+
+        self.first_headers = self.get_auth_header_for("firstuser@mail.com")
+        # self.other_headers = self.get_auth_header_for("someuser@mail.com")
+
+        db.session.flush()
+
+    def test_get_reference_request_by_response_id(self):
+
+
+        self.seed_static_data()
+        reference_req = ReferenceRequest(1, 'Mr', 'John', 'Snow', 'Supervisor', 'common@email.com')
+        # reference_req2 = ReferenceRequest(1, 'Mrs', 'John', 'Jones', 'Manager', 'john@email.com')
+
+        params = {'event_id': 1, 'application_form_id': 1, 'user_id:': 5, 'sort_column' : 'review_response_id'} # confirm params, check ids
+        reference_request_repository.create(reference_req)
+        responses = [
+            self.app.get('/api/v1/review', headers=self.get_auth_header_for('r1@r.com'), data=params),
+            self.app.get('/api/v1/reference-request/list', data={'response_id': 1}, headers=self.first_headers)
+        ]
+
+        # Response(1, 5)
+        # reference_request_repository.create(reference_req2)
+
+        data = json.loads(responses.data)

```

Comment:
As it currently stands, responses is a list, so it doesn't have a `data` property. But based on my previous comment it doesn't need to be a list anyway and then this will work :)